Jumpscale/builder/runtimes/BuilderRust.py

```diff
@@ -8,16 +8,48 @@ class BuilderRust(j.builder.system._BaseClass):
     DOWNLOAD_URL = "https://sh.rustup.rs"
 
     def _init(self):
-        self.DIR_BUILD = self._replace("{DIR_VAR}/build/rust")
+        self.DIR_BUILD = "/root/.cargo/bin"
 
     @builder_method()
-    def install(self):
+    def build(self):
         self.profile_sandbox_select()
 
         # Will download and run the correct version of rustup-init executable for your platform
         self._execute(
             "curl {} -sSf | sh -s -- -y".format(self.DOWNLOAD_URL)
         )
+        self._copy(self.DIR_BUILD, "{DIR_BIN}")
+
         self._execute("source $HOME/.cargo/env")
-        self._execute("echo  'export PATH='$HOME/.cargo/bin:$PATH'' >> ~/.bashrc ")
+        self._execute("echo  'export PATH='/root/.cargo/bin:$PATH'' >> ~/.bashrc ")
+
+    @builder_method()
+    def install(self):
```

Comment:
should here copy binary from builder path to ```/sandbox/bin```

Reply:
`self._copy(self.DIR_BUILD, "{DIR_BIN}")` ?

Reply:
yes